src/csv.hpp

```diff
@@ -0,0 +1,1248 @@
+// Copyright: (2012-2015) Ben Strasser <code@ben-strasser.net>
```

Comment:
Conversation in osrm-backend https://github.com/Project-OSRM/osrm-backend/issues/3974